src/index.tsx

```diff
@@ -200,7 +201,7 @@ export class App extends React.Component<{}, AppState> {
     e.preventDefault();
   };
 
-  public componentDidMount() {
+  public async componentDidMount() {
```

Comment:
Woow, mind blown, had no idea you could make componentDidMount async

Reply:
You can make `componentDidMount` async, for React is the same, because it will not await for the resolution of the method.